components/sidebar/SidebarElement.vue

```diff
@@ -10,25 +10,29 @@ const { data: collections } = useTrackCollections();
 
 <template>
   <aside
-    class="min-w-[300px] border-r border-slate-200 bg-slate-100 p-6 sticky flex flex-col gap-8"
+    class="min-w-[300px] max-h-[100vh] border-r border-slate-200 flex flex-col"
```

Comment:
Here you could use the `max-h-screen` class instead of the `max-h-[100vh]` class 😊 But that's fine